Library/Homebrew/formulary.rb

```diff
@@ -251,4 +253,19 @@ def self.tap_paths(name)
       ]).detect(&:file?)
     end.compact
   end
+
+  def self.find_with_priority(ref, spec=:stable)
+    possible_pinned_tap_formulae = tap_paths(ref, Dir["#{HOMEBREW_LIBRARY}/PinnedTaps/*/*/"]).map(&:realpath)
```

Comment:
`FormulaLoader#initialize` does a `path.resolved_path`, which does not work for this case because we have a symlink on the folder containing the formula file instead of the file itself.

Shall I change `FormulaLoader#initialize` from `resolved_path` to `realpath`? I'm not 100% sure if it is safe here.